# Translations template for heat.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the heat project.
#
# Translators:
# Andreas Jaeger <jaegerandi@gmail.com>, 2014
# Ettore Atalan <atalanttore@googlemail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: Heat\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-03-07 06:04+0000\n"
"PO-Revision-Date: 2015-02-25 19:06+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: German (http://www.transifex.com/projects/p/heat/language/"
"de/)\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: heat/api/aws/ec2token.py:161
msgid "Ec2Token authorization failed, no auth_uri specified in config file"
msgstr ""

#: heat/api/aws/utils.py:99
#, python-format
msgid "Request does not contain %s parameter!"
msgstr ""

#: heat/api/cloudwatch/watch.py:253
msgid "Request does not contain required MetricData"
msgstr ""

#: heat/common/context.py:202
msgid "Keystone v3 API connection failed, no password trust or auth_token!"
msgstr ""

#: heat/common/exception.py:114
msgid "Exception in string format operation"
msgstr "Ausnahme bei Zeichenfolgeformatoperation"

#: heat/common/heat_keystoneclient.py:135
msgid "Admin client authentication failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:156
msgid "Domain admin client authentication failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:170
msgid "Keystone client authentication failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:176
msgid "trust token re-scoping failed!"
msgstr ""

#: heat/common/heat_keystoneclient.py:180
msgid "Trust impersonation failed"
msgstr ""

#: heat/common/heat_keystoneclient.py:310
#: heat/common/heat_keystoneclient.py:384
#, python-format
msgid "Failed to add user %(user)s to role %(role)s, check role exists!"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:46
msgid "failed to get lifecycle plug point classes"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:53
#, python-format
msgid "failed to instantiate stack lifecycle class %s"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:58
msgid "failed to sort lifecycle plug point classes"
msgstr ""

#: heat/common/lifecycle_plugin_utils.py:111
#, python-format
msgid "%(opname) %(ci)s failed for %(a)s on %(sid)s"
msgstr ""

#: heat/common/plugin_loader.py:100
#, python-format
msgid "Failed to import module %s"
msgstr ""

#: heat/common/wsgi.py:265
msgid "SIGTERM received"
msgstr "SIGTERM erhalten"

#: heat/common/wsgi.py:274
msgid "SIGHUP received"
msgstr "SIGHUP empfangen"

#: heat/common/wsgi.py:302
#, python-format
msgid "Removing dead child %s"
msgstr "Entfernen von inaktivem untergeordnetem Element %s"

#: heat/common/wsgi.py:660
#, python-format
msgid "Exception handling resource: %s"
msgstr "Ausnahmebehandlungsressource: %s"

#: heat/common/wsgi.py:682
#, python-format
msgid "Returning %(code)s to user: %(explanation)s"
msgstr ""

#: heat/common/wsgi.py:756
#, python-format
msgid "Unexpected error occurred serving API: %s"
msgstr ""

#: heat/engine/api.py:41
msgid "Timeout conversion failed"
msgstr ""

#: heat/engine/api.py:299
msgid "Unexpected number of keys in watch_data.data!"
msgstr ""

#: heat/engine/environment.py:492
#, python-format
msgid "Failed to read %s"
msgstr ""

#: heat/engine/environment.py:504
#, python-format
msgid "Failed to parse %(file_path)s"
msgstr ""

#: heat/engine/environment.py:508
#, python-format
msgid "Failed to read %(file_path)s"
msgstr ""

#: heat/engine/plugin_manager.py:97
#, python-format
msgid "Failed to load %(mapping_name)s from %(module)s"
msgstr ""

#: heat/engine/plugin_manager.py:104
#, python-format
msgid "Invalid type for %(mapping_name)s from %(module)s"
msgstr ""

#: heat/engine/resource.py:459
msgid "Error marking resource as failed"
msgstr ""

#: heat/engine/resource.py:912 heat/engine/resource.py:939
#, python-format
msgid "DB error %s"
msgstr "Datenbankfehler %s"

#: heat/engine/resource.py:1063
#, python-format
msgid "signal %(name)s : %(msg)s"
msgstr "Signal %(name)s : %(msg)s"

#: heat/engine/service.py:356
#, python-format
msgid "Failed to stop engine service, %s"
msgstr ""

#: heat/engine/service.py:1374
msgid "Filtering by namespace/metric not yet supported"
msgstr ""

#: heat/engine/service_stack_watch.py:69
#, python-format
msgid "Unable to retrieve stack %s for periodic task"
msgstr ""

#: heat/engine/stack.py:520
#, python-format
msgid "Exception: %s"
msgstr "Ausnahmesituation: %s"

#: heat/engine/stack.py:770
#, python-format
msgid "Unexpected action %s passed to update!"
msgstr ""

#: heat/engine/stack.py:1000
#, python-format
msgid "Unexpected action %s passed to delete!"
msgstr ""

#: heat/engine/stack.py:1184
#, python-format
msgid "Resource %(name)s delete failed: %(ex)s"
msgstr ""

#: heat/engine/stack.py:1193
#, python-format
msgid "Resource %(name)s create failed: %(ex)s"
msgstr ""

#: heat/engine/worker.py:76
#, python-format
msgid "WorkerService is failed to stop, %s"
msgstr ""

#: heat/engine/clients/__init__.py:143
#, python-format
msgid "Invalid cloud_backend setting in heat.conf detected  - %s"
msgstr ""

#: heat/engine/resources/wait_condition.py:72
#, python-format
msgid "Metadata failed validation for %s"
msgstr ""

#: heat/engine/resources/aws/autoscaling/autoscaling_group.py:341
msgid "Failed sending error notification"
msgstr ""

#: heat/engine/resources/aws/ec2/eip.py:113
msgid ""
"No default floating IP pool configured. Set 'default_floating_pool' in nova."
"conf."
msgstr ""

#: heat/engine/resources/aws/ec2/eip.py:327
#: heat/engine/resources/aws/ec2/eip.py:350
msgid "Port not specified."
msgstr "Port nicht angegeben."

#: heat/engine/resources/openstack/heat/remote_stack.py:148
#, python-format
msgid "exception: %s"
msgstr ""

#: heat/engine/resources/openstack/nova/nova_floatingip.py:78
msgid ""
"Could not allocate floating IP. Probably there is no default floating IP "
"pool is configured."
msgstr ""

#: heat/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "in Schleifenaufruf mit festgelegter Dauer"

#: heat/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "in dynamischen Schleifenaufruf"

#: heat/openstack/common/policy.py:563 heat/openstack/common/policy.py:843
#, python-format
msgid "Failed to understand rule %s"
msgstr "Regel %s konnte nicht verstanden werden"

#: heat/openstack/common/policy.py:573
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Kein Handler für Übereinstimmungen des Typs %s"

#: heat/openstack/common/service.py:268
msgid "Unhandled exception"
msgstr "Nicht behandelte Ausnahme"
